bson/bson_fuzz_corpus_common.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) MongoDB, Inc. 2023-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package bson
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+type TestCase struct {
```

Comment:
There is no need for this common file or to export any of this data. Can we unexport it and just duplicate the "findJSONFilesInDir" and "JsonToBytes" functions in the cmd/build-oss-fuzz-corpus/main.go ?